CDP4Addin/Settings/AddinSettings.cs

```diff
@@ -0,0 +1,22 @@
+﻿// -------------------------------------------------------------------------------------------------
+// <copyright file="AddinSettings.cs" company="RHEA System S.A.">
+//   Copyright (c) 2018-2020 RHEA System S.A.
+// </copyright>
+// -------------------------------------------------------------------------------------------------
+
+using System.Collections.Generic;
```

Comment:
as per convention, using statements go inside namespace

Reply:
issue has been fixed and the file has been moved to CDP4Composition/Plugins/Settings